### DISCUSSION TITLE
Export to clipboard

### DISCUSSION THREAD

**Discussion by synelle** [lairdubois/lairdubois-opencutlist-sketchup-extension#416] [Ideas]
Hello,
I'm using quite a lot this amazing plugin and I would like to know if it was possible to implement an "export to clipboard" function exacly like the export to csv ?

**Comment by mobilarte** [comment]
Hi,

Could you please describe in more details what the use case would be?

In the next version (4.0) there will be a preview function to display the rows/columns of data to be exported, because we added functions to customize the columns prior to exporting. Copying to clipboard from this preview may be possible, but we might have compatibility issues across version/os.

**Reply by synelle** [reply]
It would be to past directly the table into a Layout blueprint. Actualy I do it manualy using the keyboard, which implies posible typing mistakes.

**Reply by mobilarte** [reply]
This feature has been implemented in commit https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/commit/92ddef5f9d582ff962366178d8605ca472390b73 .

**Reply by synelle** [reply]
Thank-you very much !
